### PR TITLE
Index creation

### PR DESCRIPTION
**Pull request by omersolmazer**
Index page created for the inventory

### CODE REVIEW

File: app/nyu-devops-s17-inventory/static/index.html [hunk 1]
```diff
@@ -1,24 +1,43 @@
...
+              <img class = "newappIcon" src="static/images/newapp-icon.png">
+            </td>
+      <td>
+        <h1 id = "message">Inventory REST API Service</h1>
+        <p class='description'>
+        GET <a href="/inventory/products">/inventory/products</a> to see all the products.
+        <br><br>
+        GET /inventory/products/{id} retrieves a product from the DB using an ID.
+        <br><br>
+        PUT /inventory/products/{id} updates a product using an ID.
```
Comment (p2hang) [inline app/nyu-devops-s17-inventory/static/index.html]: update method only updates the quantity of a single type for a kind of product :
{
       "type":"open_box", 
        "quantity":20
}

File: app/nyu-devops-s17-inventory/static/index.html [hunk 2]
```diff
@@ -1,24 +1,43 @@
...
+        <br><br>
+        DELETE /inventory/products/{id} removes a product from the DB using an ID.
+        <br><br>
+        POST /inventory/products creates an new product.
```
Comment (p2hang) [inline app/nyu-devops-s17-inventory/static/index.html]: For create we only allocates the space for storage as initialization:

{
      "restock_level": 100
}

**Review by p2hang — COMMENTED** [review]
example data might need update

File: app/nyu-devops-s17-inventory/static/index.html (hunk 1 above)
Comment (pinocchiochang) [reply app/nyu-devops-s17-inventory/static/index.html]: Yes. Missed a argument for this

**Review by pinocchiochang — APPROVED** [review]
Very Good Job!!

**Review by BobZcao — APPROVED** [review]
Looks Good!

**Review by p2hang — APPROVED** [review]
Good job!

**Pull request merged**